### ISSUE TITLE
Keyboard malfunctions after using KeyboardSplitterXbox

### ISSUE THREAD

**Issue by dmitryartemiev** [djlastnight/KeyboardSplitterXbox#192]
### Description of the issue
After utilizing the KeyboardSplitterXbox application, my Keychron V1 keyboard experiences consistent issues. It is recognized as connected when I start my PC, yet fails to function. A system reboot is required to restore keyboard functionality. Moreover, the keyboard becomes non-responsive following a wake-up from sleep mode, necessitating another reboot.

### Steps to reproduce
1. Start the PC with the Keychron V1 keyboard connected.
2. Observe that the keyboard lights up but does not register any keystrokes.
3. Restart the PC or disconnect and reconnect the keyboard to temporarily fix the issue.
4. Put the PC to sleep and wake it up.
5. Notice that the keyboard is no longer responsive again.

### Expected behavior
The keyboard should operate normally upon system start and wake-up from sleep without requiring a reboot or reconnection.

### Actual behavior
The keyboard fails to respond to keystrokes after initial power-up and waking the PC from sleep, until the system is rebooted.

### System configuration
- **Operating System**: Windows 11 Home 22H2 22621.2715
- **KeyboardSplitterXbox version**: 2.2

### Additional details
I have also reset my keyboard to factory settings, but this has not resolved the issue. Reconnecting the keyboard in different USB ports also doesn't help.

### Possible workaround
Reloading the PC seems to temporarily resolve the issue, but this is not a viable long-term solution.

I would greatly appreciate any assistance or guidance in resolving this matter.

Thank you for your time and support.

**Comment by djlastnight** [comment]
Hey, thanks for the details on this issue. I am wondering is this caused by Windows 11?
Also does this keyboard has its own 3rd party drivers - this is important too.
Can you run this keyboard on Windows 10 with the same Keyboard Splitter version and tell us what's going on.
Note that W11 is not officially listed as supported, since I never tried it.

Thanks in advance.

**Comment by Echo545** [comment]
I came across the exact same issue today after installing the KeyboardSplitterXbox my external keyboard no longer registers any keystrokes even after factory restarting it. 
I am running Windows 10 Version 10.0.19045 Build 19045 with the Nuphy Air 75 keyboard.

**Comment by djlastnight** [comment]
ok gus, but please answer my questions, so I would know where problem is.

**Comment by dmitryartemiev** [comment]
So, as I can see yes, Keychron keybroards have its own drivers. 

I have the same issue on W10.

Here is the video showing the issue:
https://drive.google.com/file/d/1YnHw8svcIWPeyE9akSi5LwTLWVM3AnBj/view?usp=drive_link

Technically, my keyboard is connected to my monitor, and every time the monitor goes to sleep, I need to restart my computer to get the keyboard to work.

As you can see in the video, after reconnecting the keyboard also doesn't work; the PC must be restarted for keyboard to function again.

This happened after using KeyboardSplitterXbox.

**Comment by djlastnight** [comment]
ok, this means W11 is not the problem here, so we have some progress. Can you uninstall the drivers for this keyboard (and use the windows generic keyboard driver or whatever it is called) and tell whether you are encountering the same issue. I am trying to understand which is causing the issue.

**Comment by dmitryartemiev** [comment]
All drivers reinstall and keyboard factory reset was the first things I've tried so it's not solving the issue unfortunately

**Comment by djlastnight** [comment]
What you mean by reinstall. I am trying to say that you should try to use the keyboard without the drivers from its manufacturer, but use it with the Microsoft drivers. Probably you get me, but I want to be 100% sure that no 3rd party drivers are used.

**Comment by dmitryartemiev** [comment]
Yeah I got you. Just tried to uninstall keyboard drivers and let windows define the keyboard by itself but it also doesn't work

**Comment by djlastnight** [comment]
Can you try to use usb slot directly from your PC, without using any kind of hubs, etc. I am definitely having similar issue sometimes, but I was always blaming the hub.

**Comment by tahaerdogan21** [comment]
Hi guys, I tried keyboard splitter 2.3.0.0 and sometimes my keyboard stops working and it requires reboot pc with button.. Using Windows 10 and very expensive keyboard. I tried to delete interception thing and will reply the results soon.

Edit: Sometimes reconnecting the usbs of the keyboard fixes the issue but sometimes it requires reboot.

**Comment by bssrdf** [comment]
Hi, I am having the same issue on Windows 11 too. Keyboard is connected via a KVM switch.  Is there a way to uninstall all drivers or system files keyboard splitter installed? Thanks.

**Comment by sayantanmajee** [comment]
HI, i am facing the same issue. tried all the options which were discussed, nothing works. please help me with the solution.

**Comment by mjc-alex** [comment]
> Hi, I am having the same issue on Windows 11 too. Keyboard is connected via a KVM switch. Is there a way to uninstall all drivers or system files keyboard splitter installed? Thanks.

I just simply delete all the files named "keyboard splitter". I use everything to search them, and it works.

**Comment by duyy1805** [comment]
Same problem :((
